src/gwt/src/org/rstudio/studio/client/workbench/views/console/shell/assist/RCompletionManager.java

```diff
@@ -550,7 +551,8 @@ else if (keycode == 113) // F2
          
          // if we insert a '/', we're probably forming a directory --
          // pop up completions
-         if (keycode == 191 && modifier == KeyboardShortcut.NONE)
+         if (modifier == KeyboardShortcut.NONE &&
```

Comment:
Very grateful to see this check as it means we don't have to worry about the Windows issue with the key codes being incorrect!